Kbuild.in

```diff
@@ -157,7 +157,6 @@ include $(src)/stob/ut/Kbuild.sub
 include $(src)/ut/Kbuild.sub
 include $(src)/xcode/ut/Kbuild.sub
 include $(src)/be/ut/Kbuild.sub
-include $(src)/sns/ut/Kbuild.sub
```

Comment:
I don't understand this.

If Galois is removed, the whole module building in Kernel space should be disabled/removed.

So, we should modify the top-level Makefile.am to disable it.

Reply:
Hi Hua,
Full Galois is not removed yet and it will be done in parts. This PR is only about disabling the Galois function calls and respective STs. Removal / disabling of whole module from kernel space will be done in next PRs.

Reply:
OK.